bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -93,7 +93,9 @@ object BagsManager {
     }
   }
 
-  private class BagsThread(id: Int) extends Runnable {
+  private class BagsThread(id: Int) extends Thread {
+    setPriority(Thread.NORM_PRIORITY - 1)
```

Comment:
I'm not sure if there is a noticeable effect setting a thread's priority, it is also system dependent.
Why do you want to have lower priority? Are you seeing performance troubles?